README.md

```diff
@@ -3,40 +3,76 @@
   <img width=600 src="https://user-images.githubusercontent.com/5758427/231207493-62676aea-4cb9-4bb4-92b0-20309c8a933a.png">
 </p>
 <hr/>
+
 <p align="center">
-  <a href="https://docs.zama.ai/concrete"> 📒 Read documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
+  <a href="https://docs.zama.ai/concrete"> 📒 Documentation</a> | <a href="https://zama.ai/community"> 💛 Community support</a> | <a href="https://github.com/zama-ai/awesome-zama"> 📚 FHE resources</a>
 </p>
 <p align="center">
 <!-- Version badge using shields.io -->
   <a href="https://github.com/zama-ai/concrete/releases">
     <img src="https://img.shields.io/github/v/release/zama-ai/concrete?style=flat-square">
   </a>
-  <!-- Link to tutorials badge using shields.io -->
-  <a href="#license">
+  <!-- License badge -->
+  <a href="#-license">
     <img src="https://img.shields.io/badge/License-BSD--3--Clause--Clear-orange?style=flat-square">
   </a>
 <!-- Zama Bounty Program -->
   <a href="https://github.com/zama-ai/bounty-program">
```

Comment:
by the way, the color of the bounty is not _that_ nice, is it? could we use Zama's yellow?